base/applications/mspaint/common.h

```diff
@@ -10,6 +10,22 @@
 
 #pragma once
 
+#define GRIP_SIZE 3
+#define MIN_ZOOM 125
+#define MAX_ZOOM 8000
```

Comment:
With proper indentation/alignment:
```suggestion
#define GRIP_SIZE   3
#define MIN_ZOOM    125
#define MAX_ZOOM    8000
```